Fix Fluid Tanks generating static lag from trying to pull /push fluid from Items.AIR

aria1th opened the pull request:
Just added isEmpty() check
I expect inventory is not null, so please tell me if it should expect some weird Nullable cases?

aria1th committed:
Did you know this?

Tank units can just generate lags by existence

Ayutac commented:
> import net.fabricmc.fabric.api.transfer.v1.item.InventoryStorage;

is used in the method, so I guess a non-null check is sensible.

aria1th committed:
add Valid inventory check

Very modded extension can declare RebornInventory with size 0 to only process fluids without items, for that case

aria1th committed:
separate as function 

and more condition check to reduce extra lag

Commit:
Parentheses
